scripts/images/convert_qcow2_ova.py

```diff
@@ -83,7 +84,9 @@
 """
 
 template_rhel_bash = """#!/bin/bash
-subscription-manager register --force --auto-attach --username={{ rh_sub_username }} --password={{ rh_sub_password }}
```

Comment:
I think down the line we are calling subscription-manager command, even that needs to be fixed I feel..

Reply:
yes , will fix it.